apps/prairielearn/src/tests/manualGrading.test.ts

```diff
@@ -1,16 +1,17 @@
-// @ts-check
-const { assert } = require('chai');
-const cheerio = require('cheerio');
-const _ = require('lodash');
-const { step } = require('mocha-steps');
-
-const { config } = require('../lib/config');
-const fetch = require('node-fetch').default;
-const helperServer = require('./helperServer');
-const sqldb = require('@prairielearn/postgres');
+import { assert } from 'chai';
+import * as cheerio from 'cheerio';
+import * as _ from 'lodash';
+import { step } from 'mocha-steps';
+
+import { config } from '../lib/config';
+import fetch from 'node-fetch';
+import helperServer = require('./helperServer');
+import helperClient = require('./helperClient');
+import sqldb = require('@prairielearn/postgres');
+
 const sql = sqldb.loadSqlEquiv(__filename);
```

Comment:
Things like this should come after all `import` statements.

Reply:
Done in f58874f8a.